docs/examples/800_istio.adoc

```diff
@@ -0,0 +1,169 @@
+= Istio Support
+
+== Overview
+
+Coherence Operator 3.1.15 and later supports Istio 1.9.1 and later. You can run the operator and Coherence cluster managed by the operator with Istio sidecar injection enabled.  Coherence caches can be accessed from outside via Coherence*Extend, REST, and other supported Coherence clients.  
+
+== Limitations
+
+The current support for Istio has the following limitation:
+
+ * TLS between cluster members do not work in Istio environment. Coherence client can support TLS through Istio secure gateway with TLS termination to connect to Coherence cluster running inside kubernetes.  For example, you can apply the following Istio gateway and and virtual service in the namespace of the Coherence cluster:
+
+tlsGateway.yaml
+[source,bash]
+----
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: tlsgateway
+spec:
+  selector:
+    istio: ingressgateway # use istio default ingress gateway
+  servers:
+  - port:
+      number: 8043
+      name: tls
+      protocol: TLS
+    tls:
+      mode: SIMPLE
+      credentialName: "extend-credential" # must be the same as secret for the TLS key and cert
+      maxProtocolVersion: TLSV1_3
+    hosts:
+    - "*"
+----
+
+
+tlsVS.yaml
+[source,bash]
+----
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: extend
+spec:
+  hosts:
+  - "*"
+  gateways:
+  - tlsgateway
+  tcp:
+  - match:
+    route:
+    - destination:
+        port:
+          number: 20000                    # the proxy server listening port
+        host: example-cluster-proxy-proxy  # the ClusterIP service name of the proxy server
+----
+
+Then a Coherence*Extend client can connect to the proxy server via TLS protocol at port 8043.
+
+== Using the Coherence operator with Istio
+
+NOTE: These instructions assume that you are using a Kubernetes cluster with Istio installed and configured already.
+
+You can deploy the operator into a namespace which has Istio automatic sidecar injection enabled. Before installing the operator, create the namespace in which you want to run the Coherence operator and label it.
```

Comment:
I think this section needs to be done differently. It probably needs to show actual examples of installing the Operator with Istio, as there are different ways to install the Operator. Using the Helm chart you can only install into a properly labelled namespace. Using the Operator's manifest yaml files you can either install into a label namespace or use `istiojctl kube-inject`. Using the manifest files with kubectl/Kustomize you have the same options.

Thinking about it more - I'm not sure why you would put the Operator behind Istio unless the customer just has a policy that everything is behind Istio. What we're really interested in is running Coherence clusters with Istio, so maybe move running the Operator with Istio after the section about running Coherence. The main reason I can think to do this would be to secure the Operator REST endpoint behind TLS using Istio, but I don't think this will actually work. Probably need a JIRA for this.

If you install into a namespace that does not have the Istio label but you add the `sidecar.istio.io/inject=true` label to a Pod does the sidecar get injected - Istio docs suggest it should. If so then this is another Option and is also an option for installing Coherence clusters into a non-labeled namespace by adding that label to the Coherence yaml.